lib/vyasa_web/components/contexts/read.ex

```diff
@@ -218,31 +216,31 @@ defmodule VyasaWeb.Context.Read do
          ==> reading mode
      ==> prioritise urls_params
 
-
-  RENAME: init_context()
-  - initialises the
-  - it's not ALWAYS a replying to
-
   """
   def init_reply_to_context(
         %Socket{
           assigns: %{
-            session: %{sangh: %{id: sangh_id}}
+            session: %{sangh: %{id: _sangh_id}},
+            draft_reflector: %Sheaf{
+              parent_id: parent_id
```

Comment:
guard clauses work here

is_binary and _

Reply:
i dodo-ed and missed this before merging, will put this in my upcoming PR